### ISSUE TITLE
[Feature request] several sni (and ws host header, and ... ) for a node

### ISSUE THREAD

**Issue by MrVb0** [wyx2685/v2board#37]
Hello, good time, thank you for your efforts
According to what has been heard, dividing the reality traffic between several SNIs is a very convenient way to avoid IP detection and blocking.
As discussed in this topic https://github.com/XTLS/Xray-core/discussions/2450 and other topics, mci in Iran recently detects realty and quickly blocks IP.
We are looking for ways to make this diagnosis and identification more difficult for them.

I would be grateful if a possibility is added to v2board so that we can specify several sni for a node and with each subscription update, the sni of that node will change.
(Like something like this: https://github.com/hiddify/Hiddify-Manager/wiki/How-to-use-subdomains-of-a-website-in-Reality)

Also, it will be much better if this method can be used in the remaining sections such as "address" and "ws host header" or "http host header" (for example, enter several addresses in one configuration and change the address with each update) / For example, enter several ws host headers in one config and change ws host header with each update) By doing this, we can spread the traffic to some extent between different servers and prevent the creation of additional nodes.

**Comment by wyx2685** [comment]
@MrVb0 i think you want random hosts/snis when get subscribe.
hosts can be done. considering add it later
snis case some problems.

**Comment by MrVb0** [comment]
Yes, this is one of the things I need, and of course I have explained other things in https://github.com/wyx2685/v2board/issues/37#issue-2014542513.
I use Google Translate and some translations may be wrong, I apologize if I did not get my meaning right.

**Comment by wyx2685** [comment]
@MrVb0 xray can support random sni by default.
only considering go with that

**Comment by Mjeir** [comment]
من تا حالا یک بار ندیدم برنامه نویسش به درخواستها اهمیت بده و فقط هر چیزی رو خودش بخواد طراحی میکنه.
ولی بازم دمش گرم . کمک خیلی بزرگی به ما کرده.

**Comment by nikbehjati** [comment]
> من تا حالا یک بار ندیدم برنامه نویسش به درخواستها اهمیت بده و فقط هر چیزی رو خودش بخواد طراحی میکنه. ولی بازم دمش گرم . کمک خیلی بزرگی به ما کرده.

دقیقا   ...
انتهای  شروع هر بحثی  به    به همینیکه  هست ختم  میشه ...
 نکته اینجاست  به کمک  دیپ سیک   تمامی تعییراتی  دلم  خواسته رو  ازش خواستم و دم دیپسیک گرم خیلی حرفه ای  با توصیح  انحام  داده  ... و عجیب اینحاست   دوبار وسظ تغییرات     پیشنهاد و ایده  داده که  خیلی لازم  بوده و جداب .. و حتی   یه بار  در خواست تغییر  چیزی  رو کردم   در انتها    غیر از اون تغییر  دو تا  اپشن  که  دیده جالب میشه  رو اصافه کرده
